provision/initramfs/init

```diff
@@ -351,7 +351,7 @@ if [ -n "$WWPOSTSHELL" -a "$WWPOSTSHELL" != "0" ]; then
     setsid /bin/cttyhack /bin/sh
 fi
 
-if [ -x "$NEWROOT/sbin/init" ]; then
+if [ -x "$NEWROOT/sbin/init" ] || [ -h "$NEWROOT/sbin/init" -a -x "$NEWROOT`readlink $NEWROOT/sbin/init`" ]; then
```

Comment:
I agree that it is smarter to see if the target exists and executable, but to do that right would technically require a recursive loop/test as the link target could also be a link. What about just testing for the existence of `$NEWROOT/sbin/init` and trust that the VNFS knows what it needs internally?